src/components/NumericInput/index.ts

```diff
@@ -86,7 +86,7 @@ class NumericInput extends TextInput {
         this._min = args.min ?? null;
         this._max = args.max ?? null;
         this._allowNull = args.allowNull ?? false;
-        this._precision = args.precision ?? 7;
+        this._precision = args.precision ?? 2;
```

Comment:
This looks like it fixes something else. I'm wondering if this should be a separate PR?